### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by luppercut** [awawa-dev/HyperHDR#828] [General]
Hello! I'm trying to install on a Debian 11 contain on Proxmox and receiving the error below:

E: Malformed entry 1 in list file /etc/apt/sources.list.d/hyperhdr.list (Component)
E: The list of sources could not be read.

**Comment by awawa-dev** [comment]
Hi
You may have made an error when pasting the command for apt-repo? Type `cat /etc/apt/sources.list.d/hyperhdr.list` and `cat /usr/share/keyrings/hyperhdr.public.apt.gpg.key` to verify it

**Reply by luppercut** [reply]
Thank you for the reply! I copied and pasted this exactly.

type -p curl >/dev/null || sudo apt install curl -y
curl -fsSL https://awawa-dev.github.io/hyperhdr.public.apt.gpg.key | sudo dd of=/usr/share/keyrings/hyperhdr.public.apt.gpg.key \
&& sudo chmod go+r /usr/share/keyrings/hyperhdr.public.apt.gpg.key \
&& echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/hyperhdr.public.apt.gpg.key] https://awawa-dev.github.io $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hyperhdr.list > /dev/null \
&& sudo apt update \
&& sudo apt install hyperhdr -y

**Reply by grannymaster** [reply]
If anyone stumbles on this, I had a similar issue on Debian 13.

My error:

```
-bash: lsb_release: command not found
Error: Malformed entry 1 in list file /etc/apt/sources.list.d/hyperhdr.list (Component)
Error: The list of sources could not be read.
```

Fixed it with this:

```
sudo rm /etc/apt/sources.list.d/hyperhdr.list
sudo apt update && sudo apt install lsb-release -y

```

Then run the one liner from the wiki.
